runtime/calamari/src/lib.rs

```diff
@@ -123,10 +122,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("calamari"),
     impl_name: create_runtime_str!("calamari"),
     authoring_version: 2,
-    spec_version: 3300,
+    spec_version: 3402,
```

Comment:
Not sure why 3400 -> 3402 and 7 -> 9.

Reply:
`Calamari Parachain Staging` network was upgraded before to `3400`/8 and then downgraded back to `3401`/9

To apply this runtime upgrade to both mainnet and staging, we need to have this weird version